source/daplink/drag-n-drop/vfs_user.c

```diff
@@ -310,6 +312,13 @@ static uint32_t read_file_details_txt(uint32_t sector_offset, uint8_t *data, uin
     pos += util_write_string(buf + pos, "Remount count: ");
     pos += util_write_uint32(buf + pos, remount_count);
     pos += util_write_string(buf + pos, "\r\n");
+
+    // Target URL, use file_buffer as a temporary storage
+    insert_target_url(file_buffer, ARRAY_SIZE(file_buffer));
```

Comment:
Because of line 95, the file_buffer will be re/used twice which may lead to incorrect computation on some cases.